arangodb-net-standard/ViewApi/Models/ViewDetails.cs

```diff
@@ -12,6 +12,11 @@ public class ViewDetails
         /// </summary>
         public const string ArangoSearchViewType = "arangosearch";
 
+        /// <summary>
+        /// Possible value for <see cref="Type"/>
+        /// </summary>
+        public const string SearchAliasViewType = "search-alias";
```

Comment:
Minor: Should we think about creating a dedicated constant class for the view types? We could open a separate issue.

Reply:
Done